apiary.apib

```diff
@@ -1186,10 +1186,11 @@ Further information can be found in the [Developers Documentation](https://devel
             }
 
 
-## Unload Data [/v2/storage/tables/{table_id}/export]
-### Export table into CSV [GET]
+## Data Preview [/v2/storage/tables/{table_id}/data-preview]
+### Data preview [GET]
 
-This method is **DEPRECATED**, please use the [asynchronous version](#reference/tables/unload-data-asynchronously) of this method.
+Returns up to 1000 rows from table in CSV format. Rows can be filtered by various filters described below.
```

Comment:
How about using `limit` and `offset` rather than a hard 100.  could maybe put a limit on limit of 1000